data_steward/analytics/cdr_ops/clean_rdr_export_qc.py

```diff
@@ -680,10 +693,23 @@
 WITH pids_with_basics AS ( -- pids with basics data --
 SELECT
 person_id
-  FROM `{{project_id}}.{{new_rdr}}.concept_ancestor`
-  INNER JOIN `{{project_id}}.{{new_rdr}}.observation` o ON observation_concept_id = descendant_concept_id
-  INNER JOIN `{{project_id}}.{{new_rdr}}.concept` d ON d.concept_id = descendant_concept_id
-  WHERE ancestor_concept_id = 1586134
+FROM `{{project_id}}.{{new_rdr}}.concept_ancestor`
```

Comment:
@brendagutman can you also update the description of this check by adding context on why the query is extended.

Reply:
@ksdkalluri I extended this check to align with logic used in the cleaning rule.  This way the check does not fail due to differing criteria. The CR should have the most accurate/current criteria to define a person with the Basics.

 I'm not sure why the CR was written this way but I can look into it and put more in the description if you'd like.

Reply:
Now I remember this makes sense.